cosi/flatcosi/mod.go

```diff
@@ -93,7 +103,8 @@ type flatActor struct {
 	reactor cosi.Reactor
 }
 
-// Sign implements cosi.Actor. It returns the collective signature of the block.
+// Sign implements cosi.Actor. It returns the collective signature of the
+// message if every participant returns its signature.
```

Comment:
a threshold, no ?

Reply:
No this implementation is oriented to be simple, there is no support for a mask and all that stuff that is needed like in the actual threshold package.